tests/test_models/stochastic.py

```diff
@@ -35,7 +37,7 @@ def utility(
     health,
     # Temporary workaround for bug described in issue #30, which requires us to pass
     # all state variables to the utility function.
-    # TODO(@timmens): Remove unused arguments once #30 is fixed.
+    # TODO(@timmens): Remove function arguments once #30 is fixed (re-use "utility").
```

Comment:
```suggestion
    # TODO(@timmens): Remove function once #30 is fixed (re-use "utility").
```

Reply:
So in stochastic.py it is the function argument, since there is only one utility function. In deterministic.py it is the entire function. But I will remove the "(re-use 'utility')" from stochastic.py